File: 250719/중앙값 구하기/find-the-median.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int numA = sc.nextInt();
        int numB = sc.nextInt();
        int numC = sc.nextInt();

        // 변수 정의
        int midNum;

        // 조건문
        if (numA > numB) {
            if (numB > numC) {
                midNum = numB;
            } else if (numA > numC) {
                midNum = numC;
            } else {
                midNum = numA;
            }
        } else {
            if (numA > numC) {
                midNum = numA;
            } else if (numB > numC) {
                midNum = numC;
            } else {
                midNum = numB;
            }
        }

        // 출력
        System.out.println(midNum);
    }
}

File: 250719/중앙값 구하기/find-the-median.py
# 입력
numA, numB, numC = map(int, input().split())

midNum = 0
# B A C
# 조건문
if numA > numB :
    if numB > numC :
        midNum = numB
    elif numA > numC :
        midNum = numC
    else :
        midNum = numA
else : 
    if numA > numC :
        midNum = numA
    elif numB > numC :
        midNum = numC
    else :
        midNum = numB

print(midNum)